governance/program/src/state/program_metadata.rs

```diff
@@ -0,0 +1,77 @@
+//! ProgramMetadata Account
+
+use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
+use solana_program::{
+    account_info::AccountInfo, clock::Slot, program_error::ProgramError,
+    program_pack::IsInitialized, pubkey::Pubkey,
+};
+use spl_governance_tools::account::{get_account_data, AccountMaxSize};
+
+use crate::state::enums::GovernanceAccountType;
+
+/// Program metadata account. It stores information about the particular SPL-Governance program instance
+#[repr(C)]
+#[derive(Clone, Debug, PartialEq, BorshDeserialize, BorshSerialize, BorshSchema)]
+pub struct ProgramMetadata {
+    /// Governance account type
+    pub account_type: GovernanceAccountType,
+
+    /// The slot when the metadata was captured
+    pub updated_at: Slot,
+
+    /// The version of the program
+    pub version: String,
```

Comment:
You may want to actually use something fixed-length here instead, or comment about the maximum supported size

Reply:
The version is variable format so I decided to keep it as string and added the max size comment